apps/challenges/utils.py

```diff
@@ -6,3 +8,9 @@
 get_challenge_model = get_model_object(Challenge)
 
 get_challenge_phase_model = get_model_object(ChallengePhase)
+
+
+def get_file_content(file_path, mode):
```

Comment:
Actually earlier we were reading all the data from the yaml file like `challenge_description` , `terms_and_conditions` etc. but as this content will be long, the yaml file configuration will not look good. So, now we are reading this content also from the different files, that's why I created this utility function to remove the redundancy in code. Please tell if I am not clear somewhere.